megamek/src/megamek/client/ui/baseComponents/AbstractSplitPane.java

```diff
@@ -18,16 +18,16 @@
  */
 package megamek.client.ui.baseComponents;
 
+import java.awt.Component;
+import java.util.ResourceBundle;
+
+import javax.swing.JFrame;
+import javax.swing.JSplitPane;

```

Comment:
We allow wildcard imports, especially for AWT and Swing

Reply:
Otherwise they're a nightmare!

Reply:
This was done automatically by Eclipse Ctrl-Shift-O that 'organizes' imports...  I'll revert this and clean it up manually when I split this into it's own pull request.